### PR TITLE
Updates README

### PR DESCRIPTION
**Pull request by leonelgalan**
I’m bored!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,76 +6,73 @@
...
 
 ```ruby
-gem "cardboard_cms"
+gem 'cardboard_cms'
 ```
 
-And run `bundle install`.
-
-Run the generator to install cardboard and it's migrations:
+Run bundler and the the generator to install cardboard and it's migrations:
```
Comment (greg5green) [inline README.md]: too many `the`s together here.

File: README.md [hunk 2]
```diff
@@ -83,21 +80,19 @@ if image = current_page.get('intro.image1')
...
+file.format                     # => :pdf
+file.name                       # => "some.pdf"
 number_to_human_size(file.size) # => "486 KB"
 ```
 
-
 ## Create Pages
-To add pages to cardboard edit `config/cardboard.yml`. See an sample `cardboard.yml` in [https://github.com/smashingboxes/cardboard/blob/master/test/dummy/config/cardboard.yml](https://github.com/smashingboxes/cardboard/blob/master/test/dummy/config/cardboard.yml)
-
+To add pages to cardboard edit _config/cardboard.yml_. See an sample [_cardboard.yml_](https://github.com/smashingboxes/cardboard/blob/master/spec/dummy/config/cardboard.yml)
```
Comment (greg5green) [inline README.md]: `See a sample`

File: README.md [hunk 3]
```diff
@@ -6,76 +6,73 @@
...
 
 ```ruby
-gem "cardboard_cms"
+gem 'cardboard_cms'
 ```
 
-And run `bundle install`.
-
-Run the generator to install cardboard and it's migrations:
+Run bundler and cardbaord's install generator
```
Comment (iandonovan) [inline README.md]: Typo! `cardbaord` should obviously be `cardboard`.

File: README.md [hunk 4]
```diff
@@ -119,86 +114,83 @@ templates:
...
+type | string | `'string'` | choose between: `'boolean'`, `'date'`, `'decimal'`, `'file'`, `'image'`, `'integer'`, `'rich_text'`, `'text'`, `'string'`, `'resource_link'` (needs value: resource_linked, same as the controller name ex: `'pianos'`), `'external_link'` (needs value: `'http://site.com'`)
+default | string | `nil` | set the value but don't overwrite (only set if `nil`). Put default images in _/assets/images/defaults_ and default files in _/assets/files/defaults_
+value | string | `nil` | USE ONLY FOR types `'resource_link'` and `'external_link'` (this will overwrite user input, in most cases use the **_default_** key instead)
 
 ## Create Resources
 To add an admin area for a model simply type (make sure the model exists first and migrations have been run)
-
 ```sh
-rails g cardboard:resource model_name
+rails generate cardboard:resource model_name
```
Comment (iandonovan) [inline README.md]: Thank you. Too many times have I come back to this readme to remember the name of a generator task, cmd+f `generate`, and come up empty handed.

File: README.md [hunk 5]
```diff
@@ -208,28 +200,27 @@ You can show filters on your resource index page simply by adding `cardboard_fil
...
-We use kaminari, so all you need to do is add to your index view:
+We use **kaminari**, so all you need to do is add to your index view:
 ```ruby
 = paginate @users
 ```
 
-
 #### Column sorting helper
-Cardboard's controllers inherit from a `@q` variable which gives access to the ransack gem.
+The `@q` variable which gives access to the **Ransack** gem.
```
Comment (iandonovan) [inline README.md]: Don't need the word "which" here.

**Comment by iandonovan** [comment]
Looks good, killer.

**Comment by elfassy** [comment]
did we want to change `repeatable: true` to `repeatable: slide` in the example? I don't remember if we merged the pull request for this

**Pull request merged**